src/Analyzers/CSharp/Tests/NewLines/EmbeddedStatementPlacement/EmbeddedStatementPlacementTests.cs

```diff
@@ -336,4 +337,39 @@ void M()
             Options = { { CSharpCodeStyleOptions.AllowEmbeddedStatementsOnSameLine, CodeStyleOption2.FalseWithSuggestionEnforcement } }
         }.RunAsync();
     }
+
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/66017")]
+    public async Task SwitchFollowedByEmptyStatement()
+    {
+        var source = """
+            class TestClass
+            {
+                void M()
+                {
+                    switch (0)
+                    {
+                    }[|;|]
+                }
+            }
+            """;
+        var fixedCode = """
+            class TestClass
+            {
+                void M()
+                {
+                    switch (0)
+                    {
+                    }
+
```

Comment:
Why does the formatter add an empty line instead of just moving the semicolon to the next line?

Reply:
we already have a rule about spacing out statements once the statements are on separate lines.  so this then activates that.